cli/cobra.go

```diff
@@ -205,6 +206,21 @@ func isExperimental(cmd *cobra.Command) bool {
 	return experimental
 }
 
+func displayHelpLink(cmd *cobra.Command) bool {
+	hideGuides := os.Getenv("DOCKER_HIDE_HELP_GUIDES") != ""
```

Comment:
Not sure if we need this; I think overall adding content to the `help` output should be ok

Reply:
cc @nebuk89 can we drop this requirement ? (I agree this doesn't seem too invasive)

Reply:
removed this env var check with other changes (comment below), we can add it back later if needed

Reply:
let's leave it out for now... I agree that it's good practice but this kind of feels like something help has just been missing rather than something very pushy :)